q2_diversity/plugin_setup.py

```diff
@@ -394,7 +445,7 @@
         'exclude_ids': Bool
     },
     outputs=[
-        ('filtered_distance_matrix', DistanceMatrix)
+        ('filtered_distance_matrix', DistanceMatrix),
```

Comment:
Style comment - when someone run `git blame` and sees that you changed this line in this PR it might be a little confusing to them, since this PR doesn't have anything to do with filtering distances matrices.

Reply:
Thanks - overlooked that typo. Removed the errant comma.